src/ui/public/visualize/loader/visualize_data_loader.ts

```diff
@@ -71,74 +67,43 @@ export class VisualizeDataLoader {
   }
 
   public async fetch(params: RequestHandlerParams): Promise<VisResponseData | void> {
-    this.vis.filters = { timeRange: params.timeRange };
-    this.vis.requestError = undefined;
-    this.vis.showRequestError = false;
```

Comment:
I ended up removing these lines since they could be added in embedded visualize handler, so that all the error-related stuff lives in one place.